src/main/java/com/metamx/metrics/JvmMonitor.java

```diff
@@ -156,10 +53,23 @@ public JvmMonitor(Map<String, String[]> dimensions)
   @Override
   public boolean doMonitor(ServiceEmitter emitter)
   {
+    // jvm/mem
```

Comment:
IMO comments in this method are no longer needed, because the called method names speak for themselves perfectly.